### ISSUE TITLE
Create conditions errors in code

### ISSUE THREAD

**Issue by manumonti** [nucypher/nucypher-ts#142]
I'm not sure if this is a documentation issue or a nucypher-ts bug.

If we check [Conditions.RpcCondition docs](https://docs.threshold.network/app-development/threshold-access-control-tac/references/conditions#conditions.rpccondition), we see there are two ways of creating this condition. But both of them return errors if used as described in docs:

1. Using RpcCondition:
```js
const const rpc = new Conditions.RpcCondition({
  chain: 1,
  method: 'eth_getBalance',
  parameters: [
      ":userAddress",
      "latest"
    ],
  returnValueTest: {
    comparator: ">=",
    value: 10000000000000
  }
});
```

2. Using Condition:
```js
const rpcConfig = {
  contractAddress: '',
  standardContractType: '',
  chain: 1,
  method: 'eth_getBalance',
  parameters: [':userAddress', 'latest'],
  returnValueTest: {
    comparator: '>=',
    value: 10000000000000,
  },
};
const rpc = Conditions.Condition(rpcConfig);
```

If we use in the code the former one, we get an error:

> "returnValueTest.value" must be a string

This error is fixed if we pass the Object key `value` as string instead of as number:

```js
  returnValueTest: {
    comparator: ">=",
    value: "10000000000000"
  }
  ```
  
  However, in the latter option, we can use both, string and number types, for the key `value`. 
  
  **Should we homogenize this behavior by allowing only one type (string or number)?**
  
  Another issue is that the latter option returns error:
  
  > Error: Request failed with status code 500
  
  It only works if we delete the two first keys of the config object:
  
  ```js
  const rpcConfig = {
    contractAddress: '', // DELETE THIS
    standardContractType: '', // DELETE THIS
    chain: 1,
    method: 'eth_getBalance',
    parameters: [':userAddress', 'latest'],
    returnValueTest: {
      comparator: '>=',
      value: 10000000000000,
    },
};
  ```
  
Also, we have the same problems in [Conditions.TimelockCondition](https://docs.threshold.network/app-development/threshold-access-control-tac/references/conditions#conditions.timelockcondition).

**Comment by manumonti** [comment]
> Nice, this is documentation error, thank you
> 
> Although I would've absolutely expected you to be able to pass the return value as an int - do you have the exact error? I'm wondering if it's `nucypher-ts` or `nucypher`

Yes, here it is:

```
conditions.js?eafd:99 Uncaught Error: "returnValueTest.value" must be a string
    at TimelockCondition.toObj (conditions.js?eafd:99:1)
    at eval (conditions.js?eafd:54:1)
    at Array.map (<anonymous>)
    at ConditionSet.toList (conditions.js?eafd:53:1)
    at ConditionSet.toJson (conditions.js?eafd:65:1)
    at ConditionSet.toWASMConditions (conditions.js?eafd:71:34)
    at Enrico.encryptMessage (enrico.js?7a1a:17:110)
    at encrypt (Encrypt.tsx?f69e:65:37)
    at HTMLUnknownElement.callCallback (react-dom.development.js?61bb:3945:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?61bb:3994:1)

react-dom.development.js?61bb:4091 Uncaught Error: "returnValueTest.value" must be a string
    at TimelockCondition.toObj (conditions.js?eafd:99:1)
    at eval (conditions.js?eafd:54:1)
    at Array.map (<anonymous>)
    at ConditionSet.toList (conditions.js?eafd:53:1)
    at ConditionSet.toJson (conditions.js?eafd:65:1)
    at ConditionSet.toWASMConditions (conditions.js?eafd:71:34)
    at Enrico.encryptMessage (enrico.js?7a1a:17:110)
    at encrypt (Encrypt.tsx?f69e:65:37)
    at HTMLUnknownElement.callCallback (react-dom.development.js?61bb:3945:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?61bb:3994:1)
```

**Comment by piotr-roslaniec** [comment]
Perhaps we should have a seperate test suite in `nucypher-ts`, `test/docs`, where we keep code examples to be used in documentation. Then, we could just copy&paste them with a high degree of confidence instead of doing a QA on them every time we update the API.

**Comment by derekpierre** [comment]
@manumonti is this part of the docs work you did to ensure that the code used in docs is always correct?

**Comment by piotr-roslaniec** [comment]
Closed by #147
